plugins/opa/pkg/kube/files.go

```diff
@@ -74,7 +75,10 @@ func SetFileClient(objects []map[string]interface{}) *Client {
 		}
 		_, err = dynamic.Resource(gvr).Namespace(namespace).Create(context.Background(), &unstructure, metav1.CreateOptions{})
 		if err != nil {
-			panic(err)
+			statusError, ok := err.(*kubeerrors.StatusError)
+			if !ok || statusError.ErrStatus.Reason != metav1.StatusReasonAlreadyExists {
```

Comment:
I think we need to do a better job of handling this case. As-is, the second file won't be scanned, and the user won't get any feedback to that effect. I'd rather `panic` and not support this case than let a failure slip through.

Assuming it's not a trivial fix, this is probably worth breaking out into its own PR

Reply:
The second file will be scanned. This only affects the in-memory kubernetes cluster that an OPA policy could potentially use (but I can't see it being a common thing in the CI context anyways). Assuming most of those checks are just checking for the existence of something this actually feels like a safe way to handle it to me.

Reply:
a panic here really doesn't feel appropriate. But we could easily log it. Or something like that. I'm okay with breaking this into a different PR. I needed it in here to test out with the Charts repo though

Reply:
Ohh I understand now. OK, yeah, I think this is good then.